src/components/Layout.js

```diff
@@ -1,11 +1,13 @@
 import Footer from './Footer';
 import Header from './Header';
+import Navbar from './Navbar/index';
 
 export default function Layout({children}) {
 	return (
 		<>
 			<Header />
 			<main>{children}</main>
+			<Navbar />
```

Comment:
If see it correctly, the components  <Footer  / > and <Header /> are empty. Please remove them.